test/spec/components/TextArea.spec.js

```diff
@@ -458,7 +460,9 @@ GEHTS
       const initialHeight = input.clientHeight;
 
       // then
-      expect(initialHeight).to.be.greaterThan(60);
+      waitFor(() => {
```

Comment:
Note that the test won't wait for the result as it exits right after the `waitFor` call. So the test will pass even though the callback may fail. We should either `await` the result or return the Promise from the test.

```suggestion
      return waitFor(() => {
```